ramin/data_type/data_type.js

```diff
@@ -0,0 +1,148 @@
+//number
+
+let num1 = 10;
+let num2 = 10.5;
+let num3 = 5/0;
+let num4 = -5/0;
+let num5 =  3e5;
+let num6 = 16546453531641654694964894949498978984n;
+let num7 = "abc"/3;
+
+//string
+
+let s1 = "salam 1";
+let s2 = 'salam 2';
+let s3 = `salam 3 ${5}`;
+
```

Comment:
**[High]**: We have other types of typing in JavaScript as well.

- bigint
- undefined
- boolean
- function
- symbol

Reply:
Also, check this:
typeof null

Reply:
yes we have.
but you write string and number i think you want this two type.

Reply:
please add theme